src/sql/workbench/browser/modal/modal.ts

```diff
@@ -345,8 +345,6 @@ export abstract class Modal extends Disposable implements IThemable {
 			this._modalBodySection.querySelectorAll(tabbableElementsQuerySelector) :
 			this._bodyContainer.querySelectorAll(tabbableElementsQuerySelector);
 
-		this._focusedElementBeforeOpen = <HTMLElement>document.activeElement;
```

Comment:
this method might be called multiple time within the lifecycle of the dialog, i am moving it to the show method which is only called once.